sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 #### Features Added
 - GA release of integrated cache functionality. For more information on integrated cache please see [Azure Cosmos DB integrated cache](https://docs.microsoft.com/azure/cosmos-db/integrated-cache).
+- GA release of analytical ttl replace functionality. For more information on analytical ttl please see [Azure Cosmos DB analytical store](https://docs.microsoft.com/azure/cosmos-db/analytical-store-introduction).
```

Comment:
no need to mark this as GA - GA is when we release functionality that was previously in beta. 
```suggestion
- Added ability to replace analytical ttl on containers. For more information on analytical ttl please see [Azure Cosmos DB analytical store](https://docs.microsoft.com/azure/cosmos-db/analytical-store-introduction).
```

Reply:
got ya